--- origsrc/metacity-2.28.0/src/Makefile.am	2009-09-08 15:55:35.000000000 -0500
+++ src/metacity-2.28.0/src/Makefile.am	2009-10-29 09:51:23.566449200 -0500
@@ -139,7 +139,7 @@ schema_bindings_SOURCES =       \
 	metacity.schemas.in.in
 
 schema_bindings_LDADD = @METACITY_LIBS@
-metacity.schemas.in: schema_bindings ${srcdir}/metacity.schemas.in.in
+metacity.schemas.in: schema_bindings$(EXEEXT) ${srcdir}/metacity.schemas.in.in
 	@echo Generating keybinding schemas... ${srcdir}/metacity.schemas.in.in
 	${builddir}/schema_bindings ${srcdir}/metacity.schemas.in.in ${builddir}/metacity.schemas.in
 
