--- origsrc/metacity-2.22.0/src/core/compositor.c	2008-03-09 16:49:15.000000000 -0500
+++ src/metacity-2.22.0/src/core/compositor.c	2008-06-01 10:51:21.671875000 -0500
@@ -38,11 +38,14 @@
 #include "xprops.h"
 #include <X11/Xatom.h>
 #include <X11/extensions/shape.h>
-#include <X11/extensions/Xcomposite.h>
 #include <X11/extensions/Xdamage.h>
 #include <X11/extensions/Xfixes.h>
 #include <X11/extensions/Xrender.h>
 
+#ifdef HAVE_COMPOSITE_EXTENSIONS
+
+#include <X11/extensions/Xcomposite.h>
+
 #if COMPOSITE_MAJOR > 0 || COMPOSITE_MINOR >= 2
 #define HAVE_NAME_WINDOW_PIXMAP 1
 #endif
@@ -55,7 +58,6 @@
 
 #define USE_IDLE_REPAINT 1
 
-#ifdef HAVE_COMPOSITE_EXTENSIONS
 static inline gboolean
 composite_at_least_version (MetaDisplay *display,
                             int maj, int minor)
